### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by IvanKiral**
### Motivation

Which issue does this fix? Fixes #`issue number`

Migration and rebranding

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

**Review by Simply007 — CHANGES_REQUESTED** [review]
Notes:
- [x] package.json - rename name of the packages (remove `kontent-` from package name)
- [ ] Remove GT-Walfheim and use OS font from the guide
- [ ] (optional) take a look to packages (try at least minor versions bump)

Great work with the Store rewrite! 👍🏼

**Comment by Simply007** [comment]
Upgrade will be done in #102

**Review by Simply007 — CHANGES_REQUESTED** [review]
- [ ] Let's unify the configuration page with new visuals (logo colors)
- [ ] Add redirects to netlify: https://github.com/kontent-ai/sample-app-react/blob/master/netlify.toml
- [ ] Try to unify font to WorkSans - if something does not look OK - try to adjust font-weight
- [x] Coffees filter does not work
- [ ] Brewers filter is missing
- [ ] I don't se subtitle on Home hero banner, Titles on Latest articles
- [ ] Rich text resolution for twitter widget and video is not resolved `~/en-us/articles/cf106f4e-30a4-42ef-b313-b8ea3fd3e5c5`

**Review by Simply007 — CHANGES_REQUESTED** [review]
If i go to some page and back - i don't see the cafes:

recording

If I go to https://deploy-preview-101--kontent-sample-app-vue.netlify.app/es-ES/articles/117cdfae-52cf-4885-b271-66aef6825612 (ES language) the link in text does not preserve nor load currect language varian of the coffee (link is `https://deploy-preview-101--kontent-sample-app-vue.netlify.app/coffees/kenya-gakuyuni-aa`)

**Comment by IvanKiral** [comment]
I agree with the first one, there was one additional condition which made this error. The second one might be due to fallbacking into English language due to not published content in ES language.

**Comment by Simply007** [comment]
> I agree with the first one, there was one additional condition which made this error. The second one might be due to fallbacking into English language due to not published content in ES language.

In that case, there should be ˙https://deploy-preview-101--kontent-sample-app-vue.netlify.app/en-US/coffees/kenya-gakuyuni-aa˙ (with language codename right). The thing is we don't want to have the same content on a different URLs. Not sure about the fallback - If we want to change the language prefix - but is should be present.

**Review by Simply007 — CHANGES_REQUESTED** [review]
Everything, but language behavior is fine.

- [x] lowercase language prefixes
- [ ] default route should enforce prefix `/articles` -> `/en-us/articles` 
- [ ] If a fallback is used enforce the fallback language right to the prefix and basically change it
- [ ] Describe the behavior in the README -> at least brief - to be ready

**Review by Simply007 — CHANGES_REQUESTED** [review]
- [ ] Homepage link for About us has a "More Articles" label.

Haven't found anything more.

**Review by Simply007 — CHANGES_REQUESTED** [review]
Update the readme to reflect the current state.

**Comment by IvanKiral** [comment]
I have updated the readme. Added  a new paragraph about data fetching with an example, fixed grammar mistakes, and adjusted the structure of the whole readme

File: src/components/Admin/Configuration.vue [hunk 1]
```diff
@@ -189,9 +184,9 @@ export default {
     redirectToHome(newProjectId) {
```
Comment (Simply007) [inline src/components/Admin/Configuration.vue:184]: I didn't get to be redirected after generating the project.

**Review by Simply007 — COMMENTED** [review]
Great work on README.

I have one comment about configuration + I might change the name of the kk-logo.svg to something more accurate.

File: src/components/Admin/Configuration.vue (hunk 1 above)
Comment (Simply007) [reply src/components/Admin/Configuration.vue:184]: Feel free to submit it separately - I don't think your changes broke it.

**Review by Simply007 — APPROVED** [review]

**Pull request merged**